docs/react-v9/contributing/dev-workflow.md

```diff
@@ -1,4 +1,104 @@
+
+We don't have an official branch naming policy. However some developers use the following to make their branches easier to find: `user/[alias]/your-fancy-branch-name`, ie `user/johndoe/fix-that-bug`.
+
+### Building
+
+The inner development loop usually involves these steps:
+
+First, `cd` to the root folder of the repo (usually `fluentui`) and run `yarn` to install dependencies and link packages. This can take some time.
+
+```
+cd fluentui
+yarn
+```
+
+Run `yarn start` and select your start up project.
+
+```
+@fluentui/public-docsite // starts the public v8 documentation site.
+@fluentui/public-docsite-v9 // starts the internal v9 documentation storybook site.
+```
+
+Pick a specific component package if you intend to work on a v9 component directly.
+
+Most projects have hot module reloading, so file saves will trigger refreshes.
+
+### Iterating
+
+Make the changes you need, and commit along the way.
+
+```
+git status // Shows all changed files
+git add . // Stages the changed files
+git commit -m "Your brief message." // Makes the commits. You may notice a slight lag during committing as our linters work away.
+git push // Pushes your changes to your forked branch.
+```
+
+### Syncing with master
+
+It is strongly recommended that you rebase your branch onto (rather than merging with) master.
+
+```
+git checkout master // Switches to master
+git pull upstream master // Syncs your local master with the latest version of master at the origin
+git checkout your-fancy-branch // Switches to your branch
+git rebase master // Tacks your commits onto the end of master.
+```
+
+Resolve any conflicts in your editor.
+
+`git push` // Pushes your changes to your forked branch.
+
+Creating _draft_ pull requests is often an easy way to keep track of your work without it being actively reviewed by others.
+
+In other cases, such as before checking in or running tests, you may need to run a full build (or build up to a certain package):
+
+- `yarn build` - build everything. It is good the run this weekly, or anytime you start a new project off master.
+- `yarn buildto package-name` - build up to a package.
+  - `yarn buildto @fluentui/react`
+
+You can also `cd` to any package under `packages/*` and run `yarn build` to build individual things, though keep in mind that this may require dependencies to be built first (using `build --to`).
+
+### Making a pull request
+
+Make sure all your changes are committed, and run a build on the package that was updated, e.g. `yarn workspace @fluentui/react-components build` for changes to the v9 components. This will compare your changes and detect if the publicly facing API has changed, and update the right docs accordingly. Commit this change.
+
+If your changes make any changes or additions to the DOM, you may need to run `yarn update-snapshots`. Check these updates in.
```

Comment:
https://github.com/microsoft/fluentui/pull/30685#discussion_r1535719152

Reply:
How would you recommend updating snapshots otherwise?

Reply:
I think `yarn nx run @fluentui/<package>:test -u` should do it

Reply:
I whole heartedly agree that snapshots are fraught with peril and should be used very judiciously. Probably calls for a whole 'nother RFC and resultant guidance doc on our overall testing strategy.

Thanks for the command :-)